libraries/Microsoft.Bot.Connector/Teams/TeamsOperationsExtensions.cs

```diff
@@ -129,7 +129,7 @@ public static async Task<TeamsMeetingParticipant> FetchParticipantAsync(this ITe
         /// The cancellation token.
         /// </param>
         /// <returns>Information regarding which participant notifications failed.</returns>
-        public static async Task<TeamsMeetingNotificationRecipientFailureInfos> SendMeetingNotificationAsync(this ITeamsOperations operations, string meetingId, TeamsMeetingNotification notification, CancellationToken cancellationToken = default(CancellationToken))
+        public static async Task<NotificationRecipientFailureInfos> SendMeetingNotificationAsync(this ITeamsOperations operations, string meetingId, BotMeetingNotificationBase notification, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Just for my own understanding - why do we need to define the method in both of TeamsOperation & TeamsOperationExtension class?

Reply:
From what I see in BotSDK .Net, It aligns with other Teams API, like fetch team detail, fetch channel list, fech meeting info. 
In `TeamsOperation`, the method constructs the request, logs the context, and sends the request. 
In `TeamsOperationExtension`, it's a concise look, functioning as a wrapper.